persistence/test/setup_test.go

```diff
@@ -1,40 +1,24 @@
 package test
 
 import (
-	"context"
 	"encoding/hex"
 	"fmt"
-	"log"
+	"github.com/pokt-network/pocket/shared/modules"
+	sharedTest "github.com/pokt-network/pocket/shared/tests/utility_module"
```

Comment:
Can you refactor this so that the package path is `github.com/pokt-network/pocket/shared/tests/` given that they are shared test utilities?

It's weird that persistence test are importing utility test helper modules.